src/dlite-errors.h

```diff
@@ -14,10 +14,10 @@ typedef enum {
   dliteNullReferenceError,
   dliteStorageOpenError,     /*!< cannot open storage plugin */
   dliteStorageLoadError,     /*!< cannot load storage plugin */
+  dliteStorageSaveError,     /*!< cannot save storage plugin */
```

Comment:
Why did this not exist before?

Reply:
It wasn't used before, but it will be in #328.